csrankings-h.csv

```diff
@@ -552,6 +552,7 @@ Heng Yin 0001,Univ. of California - Riverside,http://www.cs.ucr.edu/~heng,_1VlI0
 Heng-Da Cheng,Utah State University,https://directory.usu.edu/aggie/152b1,NOSCHOLARPAGE
 Hengchang Liu,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/1924.htm,NOSCHOLARPAGE
 Hengjin Cai,Wuhan University,https://cs.whu.edu.cn/info/1019/2884.htm,NOSCHOLARPAGE
+Hengrui Cai,Univ. of California - Irvine,https://hengruicai.github.io,NOSCHOLARPAGE

```

Comment:
Not in CS (Statistics). Please explain how they meet the inclusion criteria.

Reply:
This department is under our school of CS. That's why I thought it passed the criteria of CS related. Do you think it is possible to include this case?

Reply:
Thanks for the clarification (this is pretty unusual!).

Reply:
Right. In general it is rare to have a school of CS :) But this is something good for us as CS people :)